### PR TITLE
Feat (made-to-stick) : rich text area, code refactor

### PR DESCRIPTION
**Pull request by samim-hossain-mondal**
- introducing rich text area toolbar for adding text notes in made to stick section
- modified styles and alignments to get a seamless ux
- show succes and error snackbars on save layout and delete note cards
- code refactor

**snapshots for reference**
Screenshot 2023-03-23 at 5 39 03 PM
Screenshot 2023-03-23 at 5 45 02 PM

### CODE REVIEW

**Review by Kiranyadav1234 — APPROVED** [review]

File: src/components/MadeToStickLayout/MadeToStick.js [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+          h: card.h,
+          type: card.type,
+          emailId: card.emailId,
+        },
+      }).catch((err) => {
+        setError(ERROR_MESSAGE.ERROR);
+        console.log(err);
+      }).then(res => {
+        setSuccess(SUCCESS_MESSAGE.SAVED);
+        console.log(res);
```
Comment (Abhishek-Kharpal) [inline src/components/MadeToStickLayout/MadeToStick.js:141]: We can remove these console logs

File: src/components/routes/MadeToStick.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { Box } from '@mui/material';
-import MadeToStickLayout from '../MadeToStickLayout';
+import MadeToStick from '../madeToStickLayout/MadeToStick';
 
 export default function AnnouncementContainer() {
```
Comment (Abhishek-Kharpal) [inline src/components/routes/MadeToStick.js:5]: We should change the name of announcementContainer to MadeToStickContainer

File: src/components/MadeToStickLayout/MadeToStick.js (hunk 1 above)
Comment (samim-hossain-mondal) [reply src/components/MadeToStickLayout/MadeToStick.js]: Removed logs !

File: src/components/routes/MadeToStick.js (hunk 2 above)
Comment (samim-hossain-mondal) [reply src/components/routes/MadeToStick.js]: Fixed !

**Pull request merged**